admin-ui/plugins/auth-server/components/Scopes/ScopeListPage.js

```diff
@@ -61,6 +64,20 @@ function ScopeListPage({ scopes, permissions, loading, dispatch }) {
 
   const tableColumns = [
     { title: `${t('fields.id')}`, field: 'id' },
+    {
+      title: `${t('fields.openid')}`,
```

Comment:
The heading should be `Clients` (not `openid`). Please move this column to 3rd position